cfme/intelligence/reports/reports.py

```diff
@@ -263,14 +265,15 @@ def delete(self, cancel=False):
             assert view.is_displayed
             view.flash.assert_no_error()
         else:
-            # This check needs because after deleting the last custom report
-            # whole "My Company (All EVM Groups)" branch in the tree will be removed.
+            # This check is needed because after deleting the last custom report,
+            # the whole "My Company (All EVM Groups)" branch in the tree will be removed.
             if custom_reports_number > 1:
                 view = self.create_view(AllCustomReportsView)
                 assert view.is_displayed
             view.flash.assert_no_error()
-            view.flash.assert_message(
-                'Report "{}": Delete successful'.format(self.menu_name))
+            if not (BZ.bugzilla.get_bug(1561779).is_opened and current_version() >= '5.9'):
```

Comment:
**Optional**

Consider moving this into the test, and removing this call from the delete method, since `flash.assert_no_error()` is already called above.